modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/search/SearchSearchRequestAssemblerImpl.java

```diff
@@ -192,6 +195,19 @@ private void _setPreference(
 		}
 	}
 
+	private void _setScroll(
+		SearchRequest searchRequest, SearchSearchRequest searchSearchRequest) {
+
+		SearchTimeValue scrollTimeValue =
```

Comment:
either make it searchTimeValue or scrollSearchTimeValue, but not scrollTimeValue